libs/community/langchain_community/document_loaders/unstructured.py

```diff
@@ -103,14 +103,14 @@ def lazy_load(self) -> Iterator[Document]:
 class UnstructuredFileLoader(UnstructuredBaseLoader):
     """Load files using `Unstructured`.
 
-    The file loader uses the
-    unstructured partition function and will automatically detect the file
-    type. You can run the loader in one of two modes: "single" and "elements".
-    If you use "single" mode, the document will be returned as a single
-    langchain Document object. If you use "elements" mode, the unstructured
-    library will split the document into elements such as Title and NarrativeText.
-    You can pass in additional unstructured kwargs after mode to apply
-    different unstructured settings.
+    The file loader uses the unstructured partition function and will automatically detect the file
+    type. You can run the loader in different modes: "single" and "elements". If you use "single"
+    mode, the document will be returned as a single langchain Document object. If you use "elements"
+    mode, the unstructured library will split the document into elements such as Title and
+    NarrativeText. In addition to these modes (which are specific to the LangChain Loaders)
+    Unstructured has its own "chunking" parameters for post-processing elements into more useful
+    "chunks" for uses cases such as Retrieval Augmented Generation (RAG). You can pass in additional
+    unstructured kwargs to configure different unstructured settings.
```

Comment:
@MthwRobinson The Loader also supports a "paged" mode. Should the docstrings be updated to mention this third mode or do we want to continue leaving it out and reference the unstructured's chunking options, which includes chunking by_page?

Reply:
I opted to leave it only mentioning the two modes and reference chunking options

Reply:
@MthwRobinson, I raised the "paged" mode question here. It seemed like an somewhat irrelevant mode to support here now that we have it via chunking. It wasn't mentioned before and "paged" isn't a functionality added in this PR, so I thought it would be less confusing to tell users two ways to do the same thing. I also did not test what happens if you define both `mode="paged"` and `chunking_strategy="by_page"`

Reply:
Even though paged wasn't added in this PR, the docstrings are out of date and don't include it so might as well clean that up while we're in here.

Since there are current users of the loader who use paged mode, my suggestion would be to emit a warning log letting the user know about `chunking_page="by_page"` and raise a `ValueError` is you set both `chunking_page="by_page"` and `mode="paged"`. I don't think we should remove that mode without having a deprecation warning at least for a while so users have a chance to migrate.